### DISCUSSION TITLE
Unknown method in Application.cs

### DISCUSSION THREAD

**Discussion by shirokikhdo** [Nice3point/RevitTemplates#20] [Q&A] [answered]
Good afternoon, please tell me where this method comes from? I looked in the documentation for the UIControlledApplication class, there is no such method.
![image](https://user-images.githubusercontent.com/85278391/149653285-acfa6ec0-c377-4e7f-8861-cdfb54fef48e.png)

**Comment by Nice3point** [comment, marked as answer]
hi, from this library https://github.com/Nice3point/RevitExtensions

The implementation of this extension can be found here:
https://github.com/Nice3point/RevitExtensions/blob/main/Nice3point.Revit.Extensions/RibbonExtensions.cs#L35

**Reply by shirokikhdo** [reply]
Thanks!
